Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,224 @@
+            int iOne, iTwo;
+
+            for (iOne = 0; iOne < peoplesBalances.Length; iOne++)
+            {
+                var peopleInfo = peoplesBalances[iOne].Split(", ");
+                var values = NumPyArrayConversion(peopleInfo[1..]);
+
+                peopleNames[iOne] = peopleInfo[0];
+
+                if (attribution == "max")
+                {
+                    cash[iOne] = values.Max();
+                }
+                else if (attribution == "min")
+                {
+                    if (values.Length <= 1) continue;
+
+                    var balance = new float[values.Length - 1];
+
+                    for (iTwo = 0; iTwo < values.Length - 1; iTwo++)
+                    {
+                        if (iTwo == values.Length - 1) continue;
+                        balance[iTwo] = values[iTwo + 1] - values[iTwo];
+                    }
+                    cash[iOne] = balance.Min();
+                }
+                else if (attribution == "rich" || attribution == "poor")
+                {
+                    cash[iOne] = values[^1];
+                }
+            }
+            var maximumCash = (attribution == "max" || attribution == "rich");
+            float cashCheck = (maximumCash) ? cash.Max() : cash.Min();
+            var listBuilder = LookForPeopleCash(peopleNames, cash, cashCheck);
+            var listAllNames = ListNames(listBuilder.ToString());
+
+            return (listAllNames, cashCheck);
+        }
+
+        private static bool PeoplePlus(StringBuilder names)
+        {
+            return (names.ToString().Contains(",") || names.ToString().Contains(" and "));
+        }
+
+        private static bool IsNullOrEmpty(string[] peoplesBalances)
+        {
+            return (peoplesBalances == null || peoplesBalances.Length == 0);
+        }
+
+        private static float[] NumPyArrayConversion(string[] numbers)
+        {
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var nullString = 0;
+            int iThree, iFour;
+
+            for (iThree = 0; iThree < numbers.Length; iThree++)
+            {
+                if (string.IsNullOrEmpty(numbers[iThree])) nullString++;
+            }
+
+            var convertList = new float[numbers.Length - nullString];
+            iFour = 0;
+
+            for (iThree = 0; iThree < numbers.Length; iThree++)
+            {
+                if (string.IsNullOrEmpty(numbers[iThree]))
+                    continue;
+
+                else if (float.TryParse(numbers[iThree], out float numeric))
+                {
+                    convertList[iFour] = numeric;
+                    iFour++;
+                    continue;
+                }
+                else
+                {
+                    convertList[iFour] = numeric;
+                    iFour++;
+                }
+            }
+            return convertList;
+        }
+
+        private static StringBuilder ListNames(string nameLine)
```

Comment:
You're not listing names, you are formatting or something like that.

Reply:
Yes formatting --> formattingPeoplesNames would be better..I think!